package.json

```diff
@@ -1302,6 +1303,7 @@
         "glob": "^7.1.6",
         "mocha": "^8.3.2",
         "prettier": "^2.2.1",
+        "sinon": "^13.0.2",
```

Comment:
can this avoided? sinon is pretty jank. And we've avoided it up to now.

Reply:
ok